files/fr/web/css/forced-color-adjust/index.html

```diff
@@ -0,0 +1,102 @@
+---
+title: forced-color-adjust
+slug: Web/CSS/forced-color-adjust
+browser-compat: css.properties.forced-color-adjust
+translation_of: 'Web/CSS/forced-color-adjust'
+---
+<div>{{CSSRef}}</div>
+
+<p>La propriété CSS <strong><code>forced-color-adjust</code></strong> permet aux auteurs de ne pas activer les modes de couleurs forcées sur certains éléments. Cela restaure la possibilité de contrôler ces valeurs en CSS.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">forced-color-adjust: auto;
+forced-color-adjust: none;
+
+/* Valeurs globales */
+forced-color-adjust: inherit;
+forced-color-adjust: initial;
+forced-color-adjust: revert;
+forced-color-adjust: unset;</pre>
+
+<p>La valeur de la propriété <code>forced-color-adjust</code> doit être l'un des mots-clés suivants.</p>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+	<dt><code>auto</code></dt>
+	<dd>En mode de couleurs forcées, les couleurs de l'élément sont définies par l'{{Glossary("user agent", "agent utilisateur")}}. C'est la valeur par défaut.</dd>
+	<dt><code><strong>none</strong></code></dt>
+	<dd>Même en mode de couleurs forcées, les couleurs de l'élément ne sont pas automatiquement définies par l'{{Glossary("user agent", "agent utilisateur")}}.</dd>
+</dl>
+
+<h2 id="usage_notes">Notes d'utilisation</h2>
+
+<p>Cette propriété ne devrait être utilisée que pour faire des modifications qui prendront en charge les pré-requis de couleurs et de contrastes. Par exemple, si vous savez que les optimisations de couleur faites par l'{{Glossary("user agent", "agent utilisateur")}} conduisent à une mauvaise expérience en mode contrasté ou en mode sombre. L'utilisation de cette propriété vous permettra alors de modifier le résultat dans ce mode pour proposer une meilleure expérience. <strong>Elle ne devrait pas être utilisée dans le but d'empêcher les choix de l'utilisateur d'être respectés.</strong>.</p>
```

Comment:
superfluous dot at the end of the paragraph

Reply:
Good point! Fixed in https://github.com/mdn/translated-content/pull/1745/commits/8c4d18619131174f5e17ffa4064c2448676eea75